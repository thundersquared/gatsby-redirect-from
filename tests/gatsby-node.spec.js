const {createPages} = require('../gatsby-node');

describe('createPages', () => {
    it('tbd', () => {
        expect(true).toMatchSnapshot()
    });
});
