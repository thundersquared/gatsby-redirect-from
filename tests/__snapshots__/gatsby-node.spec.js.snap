// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createPages tbd 1`] = `true`;
